speechAssets/SampleUtterances.txt

```diff
@@ -14,6 +14,11 @@ GetStation  when is the next train arriving at {metro center|station}
 GetStation  when is the next train arriving at {ballston|station}
 GetStation  when the next train arrives at {metro center|station}
 GetStation  when the next train arrives at {ballston|station}
+GetStationHomeSet  when is the next train arrival
```

Comment:
I feel like `GetStation` should be renamed to `GetTrainsAt` and `GetStationHomeSet` should be `GetTrainsAtHome`